plugin/src/main/java/io/jenkins/plugins/casc/Configurator.java

```diff
@@ -171,19 +171,17 @@ default CNode describe(T instance, ConfigurationContext context) throws Exceptio
         return mapping;
     }
 
-
     @CheckForNull
-    default CNode describeStruct(T instance, ConfigurationContext context) {
-        return lookupConfig(context, instance.getClass()).map(configurator-> convertToNod(context, configurator, (Describable) instance)).getOrNull();
-    }
-
-
-    @NonNull
-    default Option<Configurator<T>> lookupConfig(ConfigurationContext context, Class<?> descriptor) {
-        return Option.of(context.lookup(descriptor));
+    default CNode describeStructure(T instance, ConfigurationContext context)
```

Comment:
@casz Would I need to add this to all the Configurators :P 
eg DataBound, heteroDescribable ?  Can we not typecast and still make it work ?

Reply:
you've set a default method so you only need to add it if different behaviour is required

Reply:
Cool then it sounds good that I do not have to add it to all the configurators